src/devices/SDLJoypad/SDLJoypad.h

```diff
@@ -0,0 +1,71 @@
+/*
+ * Copyright (C) 2016 iCub Facility, Istituto Italiano di Tecnologia
+ * Authors: Andrea Ruzzenenti <andrea.ruzzenenti@iit.it>
+ * CopyPolicy: Released under the terms of the LGPLv2.1 or later, see LGPL.TXT
+ */
+
+#ifndef SDLJOYPAD_H
+#define SDLJOYPAD_H
+
+#include <yarp/dev/IJoypadController.h>
+#include <yarp/dev/DeviceDriver.h>
+#include <SDL.h>
+
+
+namespace yarp {
+    namespace dev {
+        class SDLJoypad;
+      }
+}
+
+class yarp::dev::SDLJoypad : public yarp::dev::IJoypadController,
+                             public yarp::dev::DeviceDriver
+{
+
+    SDL_Joystick* m_device;
+    unsigned int  m_buttonCount;
+    unsigned int  m_axisCount;
+    unsigned int  m_stickCount;
+    unsigned int  m_touchCount;
+    unsigned int  m_ballCount;
+    unsigned int  m_hatCount;
+
```

Comment:
IMU data are not implemented yet, right? Will it be in this device or in a separated one?